build.py

```diff
@@ -0,0 +1,1000 @@
+--directories {} \
+--directories {} \
+--description \"{}\"".format(
+     VENDOR,
+     PACKAGE_URL,
+     POSTINST_SCRIPT,
+     POSTUNINST_SCRIPT,
+     PACKAGE_LICENSE,
+     MAINTAINER,
+     LOG_DIR,
+     DATA_DIR,
+     DESCRIPTION)
+
+for f in CONFIGURATION_FILES:
+    fpm_common_args += " --config-files {}".format(f)
+
+targets = {
+    'influxdb-relay' : './main.go'
+}
+
+supported_builds = {
+    'darwin': [ "amd64" ],
+    'linux': [ "amd64", "static_amd64" ]
+}
+
+supported_packages = {
+    "darwin": [ "tar" ],
+    "linux": [ "deb", "rpm", "tar" ]
+}
+
+################
+#### InfluxDB Functions
+################
+
+def print_banner():
+    logging.info(r"""
+  ___       __ _          ___  ___     ___     _           
+ |_ _|_ _  / _| |_  ___ _|   \| _ )___| _ \___| |__ _ _  _ 
+  | || ' \|  _| | || \ \ / |) | _ \___|   / -_) / _` | || |
+ |___|_||_|_| |_|\_,_/_\_\___/|___/   |_|_\___|_\__,_|\_, |
+                                                      |__/ 
+  Build Script
+""")
+
+def create_package_fs(build_root):
+    """Create a filesystem structure to mimic the package filesystem.
+    """
+    logging.debug("Creating package filesystem at location: {}".format(build_root))
+    # Using [1:] for the path names due to them being absolute
+    # (will overwrite previous paths, per 'os.path.join' documentation)
+    dirs = [ INSTALL_ROOT_DIR[1:],
+             LOG_DIR[1:],
+             DATA_DIR[1:],
+             SCRIPT_DIR[1:],
+             CONFIG_DIR[1:],
+             LOGROTATE_DIR[1:] ]
+    for d in dirs:
+        os.makedirs(os.path.join(build_root, d))
+        os.chmod(os.path.join(build_root, d), 0o755)
+
+def package_scripts(build_root, config_only=False):
+    """Copy the necessary scripts and configuration files to the package
+    filesystem.
+    """
+    if config_only:
+        logging.debug("Copying configuration to build directory.")
+        shutil.copyfile(DEFAULT_CONFIG, os.path.join(build_root, "influxdb-relay.conf"))
+        os.chmod(os.path.join(build_root, "influxdb-relay.conf"), 0o644)
+    else:
+        logging.debug("Copying scripts and sample configuration to build directory.")
+        shutil.copyfile(INIT_SCRIPT, os.path.join(build_root, SCRIPT_DIR[1:], INIT_SCRIPT.split('/')[1]))
+        os.chmod(os.path.join(build_root, SCRIPT_DIR[1:], INIT_SCRIPT.split('/')[1]), 0o644)
+        shutil.copyfile(SYSTEMD_SCRIPT, os.path.join(build_root, SCRIPT_DIR[1:], SYSTEMD_SCRIPT.split('/')[1]))
+        os.chmod(os.path.join(build_root, SCRIPT_DIR[1:], SYSTEMD_SCRIPT.split('/')[1]), 0o644)
+        shutil.copyfile(LOGROTATE_SCRIPT, os.path.join(build_root, LOGROTATE_DIR[1:], "influxdb-relay"))
+        os.chmod(os.path.join(build_root, LOGROTATE_DIR[1:], "influxdb-relay"), 0o644)
+        shutil.copyfile(DEFAULT_CONFIG, os.path.join(build_root, CONFIG_DIR[1:], "influxdb-relay.conf"))
+        os.chmod(os.path.join(build_root, CONFIG_DIR[1:], "influxdb-relay.conf"), 0o644)
+
+def run_generate():
```

Comment:
The relay doesn't use `go generate` this looks like just a leftover from borrowing influxdb's build script